chronos_bin/src/message_processor.rs

```diff
@@ -66,13 +77,18 @@ impl MessageProcessor {
                                         HashMap::new()
                                     }
                                 };
-                                //TODO: handle empty headers
-                                // println!("checking {:?}",headers);
 
                                 node_id = Some(updated_row.readied_by.to_string());
-                                // row_id = Some(updated_row.id.to_string());
 
-                                headers.insert("readied_by".to_string(), node_id.unwrap());
+                                match node_id {
```

Comment:
- We assigned Some(...) to `node_id` and then checking it with `match`. Maybe we don't need to have data type as `Option` for this variable?
- Also, it appears we are not using `node_id` outside of match block which [starts here](https://github.com/kindredgroup/chronos/pull/9/files#diff-82eea6cfd7f5aba48d47e1b4eb2c94a36db2f316bfac135e42ebbe3d378799c5R56). Then do we need to have this variable declared so early? It looks like moving `node_id` declaration into [this loop](https://github.com/kindredgroup/chronos/pull/9/files#diff-82eea6cfd7f5aba48d47e1b4eb2c94a36db2f316bfac135e42ebbe3d378799c5R62) will simplify code.